src/client/mod.rs

```diff
@@ -369,7 +369,6 @@ impl<'a> Future for ClientBuilder<'a> {
 
             self.fut = Some(Box::pin(async move {
                 let url = Arc::new(Mutex::new(http.get_gateway().await?.url));
-
```

Comment:
This change is unrelated.

Reply:
I will undo this change. My mistake, left over from some debugging during local testing.